src/main/java/org/matsim/prepare/PrepareTransitSchedule.java

```diff
@@ -101,70 +90,43 @@ public void run(Scenario scenario, String shapeFile) {
 
     private String findFareZone(TransitStopFacility transitStopFacility, Collection<SimpleFeature> features) {
 
-        String fareZone = "";
-        Boolean stopInShapes = false;
+        // Facilities which are not whithin the fare zone shapes are located outside of vvs area and thus marked accordingly
+        String fareZone = "out";
 
         Coord homeCoord = transitStopFacility.getCoord();
         Point point = MGC.coord2Point(homeCoord);
 
-        // Automatic crs detection/ transformation might be needed to avoid errors.
-
         for (SimpleFeature feature : features ) {
 
             Geometry geometry = (Geometry) feature.getDefaultGeometry();
 
             if (geometry.contains(point)) {
```

Comment:
The geotools documentation says one should prever `geometry.convers(point)` which includes the boundary of the shape and is faster. 

If this takes very long one can also use a prepared geometry

Reply:
Switched from contains -> covers in all geometry processing routines.. (network, transit schedule attribute adjustments)